stable/search-prod/templates/aggregator-deployment.yaml

```diff
@@ -70,7 +70,7 @@ spec:
           readOnlyRootFilesystem: true
         env:
         - name: REDIS_HOST
-          value: '{{ template "search.fullname" . }}-search-redisgraph'
+          value: '{{ template "search.fullname" . }}-search-redisgraph.{{ .Values.org }}.svc'
```

Comment:
Do we really need to add `{{ .Values.org }}` ?  All our components are in the same namespace.

**Note:** I'm just looking for confirmation.  This works, but maybe it's not needed.

Reply:
We faced this error before we added the namespace(Values.org).

Error:

> E0416 19:38:22.390517       1 pool.go:95] Error connecting redis. Original error: x509: certificate is valid for search-prod-e5280-search-redisgraph.open-cluster-management.svc, search-prod-e5280-search-redisgraph.open-cluster-management.svc.cluster.local, not search-prod-e5280-search-redisgraph

So, based on this [doc](https://docs.openshift.com/container-platform/4.6/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate), we added the .org

Could we have avoided that?

Reply:
ok, looks like the new certificate service adds the namespace to the dns name.